aggregate/billing_functions/hail.py

```diff
@@ -81,18 +81,23 @@ def get_finalised_entries_for_batch(batch: dict) -> List[Dict]:
         if batch_resource.startswith('service-fee'):
             continue
 
+        attributes = batch.get('attributes', {})
         labels = {
             'dataset': dataset,
             'batch_id': str(batch_id),
             'batch_resource': batch_resource,
+            'batch_name': attributes.get('name'),
```

Comment:
Can we .pop name here to avoid it being set twice?

Reply:
Oh sure thing.

Reply:
Done!